cnd/src/config/settings.rs

```diff
@@ -95,7 +95,7 @@ pub enum AllowedOrigins {
 #[derive(Clone, Debug, PartialEq, derivative::Derivative)]
 #[derivative(Default)]
 pub struct Logging {
-    #[derivative(Default(value = "LevelFilter::Debug"))]
+    #[derivative(Default(value = "LevelFilter::Trace"))]
```

Comment:
I think `Debug` or even `Info` is fine here. We just need to put trace in our e2e config files.

Reply:
Set to `Info`, thanks.